bftengine/include/bftengine/ControlStateManager.hpp

```diff
@@ -20,6 +20,27 @@
 
 namespace bftEngine {
 
+class ControlStatePage : public concord::serialize::SerializableFactory<ControlStatePage> {
+ public:
+  int64_t seqNumToStopAt_ = 0;
```

Comment:
Please change member variables to snake_case. Similar, function parameters. https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
fixed